processor/src/main/java/org/mapstruct/ap/internal/model/AbstractMappingMethodBuilder.java

```diff
@@ -117,4 +120,15 @@ public ForgedMethodHistory getDescription() {
         return description;
     }
 
+    public List<Annotation> getMethodAnnotation() {
```

Comment:
We should change the name to `getMethodAnnotations` note the extra "s" in the end to represent plural.